doc/ext/imgconverter.rst

```diff
@@ -0,0 +1,30 @@
+.. highlight:: rest
+
+:mod:`sphinx.ext.imgconverter` -- Convert images to appropriate format for builders
```

Comment:
Right. So I need better naming for this. What about `sphinx.ext.imgconverter.imagemagick`?